### DISCUSSION TITLE
Remote NFC Switch

### DISCUSSION THREAD

**Discussion by scott2645** [FrSkyRC/ETHOS-Feedback-Community#4618] [Q&A] [answered]
Has anyone figured out how to resolve and issue with the remote NFC switch not binding? I have updated with the latest firmware, and tried another switch but cannot get either of them to bind. 
![ECE7D416-48D3-48CD-A5EF-5270EF8B075B](https://github.com/user-attachments/assets/4812483e-b40d-4e91-b672-165d127667cb)
![308A6BA5-C1CC-47D1-9BEF-251FEBA55A73](https://github.com/user-attachments/assets/08dc07e5-d16c-4f52-9544-341b93cdd9be)

**Comment by Dsrenger** [comment]
First make sure you really really have a remote NFC switch. 

If you have an old NFC Switch, it cannot be made to a remote one by flashing the firmware.

If you look closly on the board you will see a "wireless" on the board and a small antenna in the the housing. If not, you dont have a remote NFC switch.

If you have follow the following:

For binding, the switch need to be connected to the S.Port of the transmitter. NOT to receiver.

But the receiver need to be On. 
If the receiver is On AND the NFC switch is connected to the S.Port of the transmitter you should be able to bind it.

Does it help?

**Reply by scott2645** [reply]
It is new wireless switch. I did exactly as described above and still get the error.
![IMG_3264](https://github.com/user-attachments/assets/a9257295-5319-4320-ad71-2d643e742b2c)

**Comment by Dsrenger** [comment, marked as answer]
What is the receiver you are using? 

Hard to see on the picture.

**Reply by scott2645** [reply]
TDR18

**Reply by Dsrenger** [reply]
Sorry, then I have no idea. 

All remote NFC switches are working fine for me  with the TD-R18.
Perhaps somebody else has an idea

**Reply by scott2645** [reply]
I have one that has worked flawless since I put it in and have over 60 flights, but I cannot get this one to work even purchased another switch and it does the same thing. I have gone over everything that I can think of and can't figure this one out. I have it set exactly like the other one that is working so it has me stumped.
